workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen/EntryPoint.cs

```diff
@@ -109,6 +111,17 @@ private string GenerateBundle()
             return bundlePath;
         }
 
+        private Dictionary<string, HashSet<string>> GenerateFileTree()
+        {
+            return options.SchemaInputDirs
+                .AsParallel()
+                .Select(schemaDir => (schemaDir,
+                    files: Directory.GetFiles(schemaDir, "*.schema", SearchOption.AllDirectories)
+                        .Select(path => path.Substring(schemaDir.Length + 1).Replace('\\', '/'))
+                        .ToHashSet()))
+                .ToDictionary(tuple => tuple.schemaDir, tuple => tuple.files);
```

Comment:
You can collapse the `Select` and `ToDictionary` into the `ToDictionary`. We create a tuple, then deconstruct the tuple immediately. Instead we can do:

```csharp
	...
	.AsParallel()
	.ToDictionary(schemaDir => schemaDir, schemaDir => Directory.GetFiles(...).Select(...).ToHashSet());
```